actions/add-watchtower.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+cat > wtinput.json
+export WT_URI=$(jq -r '.["wt-uri"]' wtinput.json)
+rm wtinput.json
+
+export MACAROON_HEADER="Grpc-Metadata-macaroon: $(xxd -ps -u -c 1000 /root/.lnd/data/chain/bitcoin/mainnet/admin.macaroon)"
+export PUBKEY=${WT_URI%%@*}
+export ADDRESS=${WT_URI#*@}
+
+sed -n -i 'H;${x;s/^\n//;s/  - wt-uri: >-.*$/  - wt-uri: >- \n      '$WT_URI'\n&/;p;}' /root/.lnd/start9/config.yaml
+
+action_result_running="    {
+    \"version\": \"0\",
+    \"message\": \"Successfully Added Watchtower $PUBKEY\",
+    \"value\": null,
+    \"copyable\": false,
+    \"qr\": false
+}"
+
+curl --no-progress-meter -X POST --cacert /root/.lnd/tls.cert --header \
```

Comment:
Pretty sure we could run this as a js action
The xxd is going to be the most complicated and might be the reason why not?

Reply:
I installed vim, so it works now. it could be a JS action, but the faster path for me was to copy the import-umbel action.